packages/alias/src/index.ts

```diff
@@ -61,6 +79,17 @@ export default function alias(options: RollupAliasOptions = {}): Plugin {
 
   return {
     name: 'alias',
+    buildStart(inputOptions) {
+      return Promise.all(
+        [...entries, options].map(
+          ({ customResolver }) =>
+            customResolver &&
+            typeof customResolver === 'object' &&
+            typeof customResolver.buildStart === 'function' &&
+            customResolver.buildStart.call(this, inputOptions)
+        )
+      ).then(() => {});
```

Comment:
Shouldn't something be caught here?

Reply:
What do you expect to catch? If something fails, then Rollup's `buildStart` will pick it up. The empty `then` is just to satisfy the expected empty return type of `buildStart`.

Reply:
We could also do a type cast here of course, but I do not think this is performance critical.

Reply:
> just to satisfy the expected empty return type

Ah! Sorry